### PR TITLE
Cleaning Stuff

### PR DESCRIPTION
**Pull request by zhouzi**
I just cleaned some stuff regarding formatting details and the readme. I understood you wanted to learn so I am planning to help. I'll commit one update at a time so it's easier to have a precise look at what's happening.

### CODE REVIEW

File: kew.js [hunk 1]
```diff
@@ -1,45 +1,70 @@
...
+                    this.elements[i].setAttribute(attrName, attr + ' ' + attrValue);
+                }
+            }
+        },
+
+        addClass: function (className) {
+            var self = this;
+
+            self.each(function (element) {
+                element.classList.add(className)
```
Comment (thomastuts) [inline kew.js]: You're missing a semicolon here.

**Comment by thomastuts** [comment]
Just to provide some general feedback here: I feel like it's kind of obsolete to assign `var self = this;` if you're not going to use it as a reference to the context you want (i.e. inside an inner function). Makes the code a bit more verbose than it should be. I'd just stick to `this` and only use `var self = this;` when actually needed.

This also makes things more consistent since you do not assign `this` in the `remove()` and the `attr()` methods.

Lastly, it's cleaner to add platform or IDE-specific things to your [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) instead of muddying up a project's `.gitignore` (e.g. the `.idea` folder for JetBrains IDE's).

**Comment by seanballais** [comment]
I second @thomastuts. `var self = this;` is unnecessary.

**Comment by zhouzi** [comment]
Thank you guys!

You are absolutely right about the `self` syntax, it's actually a personal habit. For consistency, I prefer to avoid mixing `this` and `self` which also prevent from asking yourself "do I need to store a reference to `this`?". Also, I must admit that I find it prettier :P Anyway, I'll make the changes to avoid useless assignments ;)

Good point about the global `.gitignore` too! I'll make my life simpler and just ignore those `.idea` files globally :)

**Comment by seanballais** [comment]
@Zhouzi, place the `index.html` in a `test/` folder for classification purposes.

**Comment by RodRitter** [comment]
I like the changes and clean-up. Will need to work on the readme with regards to the node isntall. The only reason node is there is for Grunt. I'm not 100% familiar with how versioning works on projects like these but would we run Grunt (minify - kew.min.js) for each new version? or everytime a change is made?

**Comment by seanballais** [comment]
Every significant change, I think.

**Comment by RodRitter** [comment]
Ok cool

**Comment by zhouzi** [comment]
I think the "distributed" version should be compiled each time a change is made, this way it's always in sync with the source file. Also, the test/demo pages are probably going to point at the built version so if you want to try out a new feature you will need to run the tasks.

The purpose of the "Install" section was to help contributor setting up their environment. Without it, they won't be able to build the minified version and you will need to do it yourself when needed.

**Comment by RodRitter** [comment]
You're right. I've added that into the readme.

**Pull request merged**